packages/sourcecred/src/plugins/external/plugin.js

```diff
@@ -0,0 +1,203 @@
+// @flow
+
+import type {Plugin, PluginDirectoryContext} from "../../api/plugin";
+import {
+  type PluginDeclaration,
+  type PluginDeclarationJSON,
+  fromJSON as declarationFromJSON,
+} from "../../analysis/pluginDeclaration";
+import {join as pathJoin} from "path";
+import {type TaskReporter} from "../../util/taskReporter";
```

Comment:
`type` should outside the curly brackets here, if only types are in the import.

Reply:
accepted all